### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by HelenBassa**
***Routing***
routing
In scope: 
- [x] add app routing
- [x] add blank pages:
/ - welcome page
/lobby - lobby page
/game - game page
/result - result page

### CODE REVIEW

File: client/src/App.tsx [hunk 1]
```diff
@@ -1,20 +1,32 @@
...
+  const routeComponents = routes.map(({ path, component }, key) => (
+    <Route exact path={path} component={component} key={key} />
+  ));
+
+  return (
+    <div className="app-container">
+      <Header />
+
+      <TransitionGroup exit={false}>
+        <CSSTransition timeout={300} classNames="transition" key={location.key}>
```
Comment (utrennik) [inline client/src/App.tsx]: Please put the transition timeout value to config file

File: client/src/components/header/header.tsx [hunk 2]
```diff
@@ -1,12 +1,25 @@
...
 import '@styles/header.sass';
 
-export const Header = () => (
+const Header = () => (
   <header className="header">
     <div className="header-up">
-      <div className="container chat-wrapper">
-        <div className="chat-icon"></div>
+      <div className="container header-up-wrapper">
+        <ul className="nav-wrapper">
```
Comment (utrennik) [inline client/src/components/header/header.tsx]: There is no navigation by design, Is this just for a test?

File: client/src/components/pages/game-page.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import '@styles/page.sass';
```
Comment (utrennik) [inline client/src/components/pages/game-page.tsx]: Plaese put the blank pages components to separate folders and update routes.

File: client/src/components/header/header.tsx (hunk 2 above)
Comment (RealOneReal) [reply client/src/components/header/header.tsx]: Yes,in future we must delete this navbar :)

File: client/src/App.tsx (hunk 1 above)
Comment (HelenBassa) [reply client/src/App.tsx]: Fixed

File: client/src/components/header/header.tsx (hunk 2 above)
Comment (HelenBassa) [reply client/src/components/header/header.tsx]: Fixed

File: client/src/components/pages/game-page.tsx (hunk 3 above)
Comment (HelenBassa) [reply client/src/components/pages/game-page.tsx]: Fixed

**Review by RealOneReal — APPROVED** [review]

**Review by utrennik — APPROVED** [review]

**Review by aksiantsiuk — APPROVED** [review]

**Pull request merged**